From 58c4f5484f1ea7918f316e21dda43877968cef35 Mon Sep 17 00:00:00 2001
From: Daniel Schaal <farbing@web.de>
Date: Thu, 19 Jun 2014 11:38:25 +0200
Subject: [PATCH] 98systemd: also install systemd-journald-dev-log.socket

The /dev/log socket was moved to /run on systemd 214, install the
newly added socket unit.
---
 modules.d/98systemd/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index dd82ecebe4..e2efc427c7 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -107,6 +107,7 @@ install() {
         $systemdsystemunitdir/sockets.target.wants/systemd-udevd-control.socket \
         $systemdsystemunitdir/sockets.target.wants/systemd-udevd-kernel.socket \
         $systemdsystemunitdir/sockets.target.wants/systemd-journald.socket \
+        $systemdsystemunitdir/sockets.target.wants/systemd-journald-dev-log.socket \
         $systemdsystemunitdir/sysinit.target.wants/systemd-udevd.service \
         $systemdsystemunitdir/sysinit.target.wants/systemd-udev-trigger.service \
         $systemdsystemunitdir/sysinit.target.wants/kmod-static-nodes.service \
-- 
1.9.3

